monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
 
 ### _Primo_
 
-An example new RPC-Pay application is the Primo project ([repo.getmonero.org/selene/primo](https://repo.getmonero.org/selene/primo)). Primo is a protocol and software suite supporting paid website content delivery. It allows undesirable website ads to be replaced with invisible and unobtrusive hash generation. A visitor to a Primo-enabled website who does not want to see ads can opt to calculate Monero mining hashes instead.
+Un nuevo ejemplo en la aplicación de RPC-Pay es el projecto Primo ([repo.getmonero.org/selene/primo](https://repo.getmonero.org/selene/primo)). Primo es un protocolo y paquete de programas que soportan la monetización del contenido para los sitios web. Le permite a los indeseables anuncios ser reemplazados con la generación de hashes invisibles y discretos. Un visitante en un sitio web con Primo habilitado que no quiera ver anuncios puede optar por calcular hashes de minería para Monero.
 
-Primo has three components. The first is the _primo-apache_ module for use on the web server. The website owner installs this module and configures what content requires payment and how much. The web server communicates with a _monerod_ instance, which handles bookkeeping on payments of hashes and using successful hashes to collect mining revenue. The second component is the _primo-firefox_ extension. A visitor to the website wanting to use RPC-Pay will have this loaded into the Firefox browser. And the third component is the _primo-control-center_ graphical control tool. With this, the Firefox user can configure what websites receive payment.
+Primo tiene tres componentes, el primero es el módulo _primo-apache_ para usarlo en el servidor web. El dueño del sitio web instala este módulo y configura que tipo de contenido requiere de pago y de cuanto. El servidor web se comunica con la instancia del _monerod_, quien es el encargado de la contabilidad en los pagos de los hashes y de usar los hashes exitosos para recaudar las ganancias de la minería. El segundo componente es la extensión _primo-firefox_. Un visitante a un sitio web que quiera utilizar RPC-Pay tendrá esto cargado en el navegador de Firefox. Y el tercer componente es la herramienta gráfica de control _primo-control-center_. Con estos componentes el usuario puede configurar que sitios web reciben pago en Firefox. 
```

Comment:
_Primo tiene tres componentes, el primero es el módulo _primo-apache_ para **uso** en el servidor web. El dueño del sitio web instala este módulo y configura **qué** tipo de contenido requiere pago, y **cuánto.** El servidor web se comunica con **una **instancia del _monerod_, **quien se encarga**..._

And a last:

_ Con estos componentes el usuario puede configurar **qué** sitios web reciben pago en Firefox._

Reply:
Added suggestions. Agree on them. :)